bot/models/player.py

```diff
@@ -127,11 +138,18 @@ async def show_progress(self, ctx):
 
         letter_xp_string = StringUtil.format_player_xp(self.get_letter_xp())
 
+        try:
```

Comment:
Can we move this logic into the `get_level` method?

Reply:
Sure thing, good call.

Reply:
Done